src/AccessibilityInsights.VersionSwitcher/InstallationOptions.cs

```diff
@@ -12,7 +12,7 @@ namespace AccessibilityInsights.VersionSwitcher
     internal class InstallationOptions
     {
         /// <summary>
-        /// The Uri to the iweb-hosted nstaller to use
+        /// The URI to the iweb-hosted installer to use
```

Comment:
Remove the extra 'i' ?
```suggestion
        /// The URI to the web-hosted installer to use
```